generate-changelog.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import re
+import datetime
```

Comment:
The imports aren't grouped properly (or sorted).

> Imports should be grouped in the following order:
> 
>   * standard library imports
>   * related third party imports
>   * local application/library specific imports
* https://pep8.org/#imports

```python
import datetime
import re

import requests
```